src/classes/VOL_CTRL_PersonalSiteContactLookup.cls

```diff
@@ -79,7 +79,7 @@ global with sharing class VOL_CTRL_PersonalSiteContactLookup {
         
         list<Contact> listCon = VOL_SharedCode.LookupContact(contact, null);
         if (listCon == null || listCon.size() == 0) {
-            strResult = System.Label.labelContactLookupNotFound;
```

Comment:
@scottwarren-sfdo - since these are not protected we can't delete them but I think the automation will attempt to delete them and we have to reference them in a class or something to prevent that from happening.